src/material/slide-toggle/slide-toggle.scss

```diff
@@ -6,11 +6,57 @@
 @use '@material/ripple' as mdc-ripple;
 @use '../core/mdc-helpers/mdc-helpers';
 @use '../core/style/layout-common';
+@use '@material/theme/custom-properties' as mdc-custom-properties;
+@use '../core/tokens/m2/mdc/switch' as m2-mdc-switch;
+@use '../core/tokens/m2/mat/slide-toggle' as mat-slide-toggle-tokens;
+@use '../core/tokens/token-utils';
 
+@include mdc-custom-properties.configure($emit-fallback-values: false, $emit-fallback-vars: false) {
+  $mdc-switch-token-slots: m2-mdc-switch.get-token-slots();
 
-@include mdc-helpers.disable-mdc-fallback-declarations {
   @include mdc-form-field.core-styles($query: mdc-helpers.$mdc-base-styles-query);
-  @include mdc-switch.static-styles-without-ripple;
+  // Add the MDC switch static styles.
+  @include mdc-switch.static-styles-without-ripple();
+
+  .mdc-switch {
+    // Add the official slots for the MDC switch
+    @include mdc-switch-theme.theme-styles($mdc-switch-token-slots);
+
+    // The tokens are outputting handle-elevation and disabled-handle-elevation
+    // as the values for box-shadow. Rather the value that should be displayed
+    // is calculated using the tokens for elevation and the handle-shadow-color
+    // within theme-styles and is stored in new variables that end in -shadow.
+
+    // TODO(amysorto): remove these overrides once MDC emits correct value for
+    // box-shadow.
+
+    // Override box-shadow with correct values.
+    &:enabled .mdc-switch__shadow {
+      box-shadow: var(--mdc-switch-handle-elevation-shadow);
```

Comment:
Not a blocker, but for cases like this I ended up creating a custom token instead. Feel free to ignore.

Reply:
This token is a bit unique since it isn't something you pass in, its something that MDC uses two other tokens (`handle-elevation` and `handle-shadow-color`) to create. There is a bug open internally to related to this issue, currently without the override it will set `box-shadow` to the other related tokens instead. Once this is fixed, there won't be a need for this override.